markdown/chapters/entombsday/playing-the-deck.md

```diff
@@ -0,0 +1,249 @@
+
+### Infinite Loops
+
+In the event that you can’t find a way to reanimate Emrakul pre-combat, you are able to *loop* your deck to either cast Emrakul or repeatedly cast :card[Collective Brutality] to drain your opponent out 2 life at a time.
+
+To *loop* your deck:
+
+1. Draw 14 to 21 cards with Griselbrand until you can put :card[Children of Korlis] into play (either by casting or reanimating it).
+2. Sacrifice Children of Korlis to regain all of the life you’ve lost so far this turn (this effect is **cumulative**, so you can gain functionally infinite life if you *loop* Children).
+3. Using the life gain from Children, draw your entire library and begin casting Dark Rituals to generate a large amount of {B} mana. 
+4. After casting all of your Rituals, discard :card[Emrakul, the Aeons Torn] to shuffle them back in. Make sure you have at least 7 cards between your deck and graveyard, so you don't lose the game from decking.
+5. At this point you should be able to cast Emrakul and take an extra turn to attack for lethal.
+
+If for some reason you’re unable to attack (i.e. there is an :card[Ensnaring Bridge] in play), you can continue to *loop* through your deck until your deck + your graveyard add up to a multiple of 7. From there, cast and sacrifice two copies of :card[Lotus Petal] for {W}{B}, cast and sacrifice Children of Korlis, cast Dark Ritual, cast Collective Brutality for its drain mode, then discard Emrakul using either :card[Thoughtseize] or :card[Cabal Therapy] to shuffle those 7 cards back into your library. You can now demonstrate a loop that generates infinite life and infinitely drains your opponent.
+
+Notably, you can’t escalate :card[Collective Brutality] to discard Emrakul, as this will leave Collective Brutality in your graveyard after the shuffle resolves, which, as Acclimation puts it in his Tin Fins primer, “does not do the thing”.
+
+## Mulligans
+
+As a combo deck, Entombsday tends to mulligan well, but requires specific cards to function. It is able to go as low as 4 cards and still combo on turn 1: a {B} source, :card[Dark Ritual], :card[Entomb], and a reanimation spell.
+
+In general, you want to leverage the London Mulligan to find the fastest hand you can in game 1, preferably with protection. You should avoid keeping hands that just durdle and cast cantrips, as that will give your opponent time to stabilize and accumulate resources.
+
+When sideboarded into Doomsday, you want to look for hands that cast Doomsday quickly -- generally before turn 3. This typically means hands that contain either Doomsday itself or Personal Tutor and the mana to cast it. As with the game 1 plan, a protection spell is also helpful. 
+
+### Sample Hands: Keep or Mull?
```

Comment:
All sample hands display only a single land in output.

Reply:
Syntax has changed and we now should use newlines rather than list items for cards within rows

```md
:::row
- One
- Two
:::
```

Should become

```md
:::row
One
Two
:::
```